core/settings_dev.py

```diff
@@ -37,6 +37,61 @@
     )
 }
 
-EMAIL_BACKEND = "django.core.mail.backends.filebased.EmailBackend"
+# Email Sending settings
+DEFAULT_FROM_EMAIL = os.environ.get("DEFAULT_FROM_EMAIL", "rakoto.olive42@gmail.com")
```

Comment:
Let's leave the default email as the one used by the person setting up the site.